volmaster/daemon.go

```diff
@@ -632,7 +632,41 @@ func (d *DaemonConfig) handleRemove(w http.ResponseWriter, r *http.Request) {
 		return etcdRemove()
 	}
 
-	err = lock.NewDriver(d.Config).ExecuteWithMultiUseLock([]config.UseLocker{uc, snapUC}, d.Global.Timeout, func(ld *lock.Driver, ucs []config.UseLocker) error {
+	if req.Options["force"] == "true" {
```

Comment:
Could you explain what you're referring to? My understanding was that this would be supposed to ignore locking and I'm not sure exactly what locking would be appropriate in this case.